Automattic/wp-calypso
# Domains: Cleanup the old Flux whois store

Sicne we're not using this Flux store anymore (see https://github.com/Automattic/wp-calypso/pull/34688 ) it's just creating unnecessary whois REST API calls. This is a cleanup diff that should remove all that code.


#### Changes proposed in this Pull Request

* Remove the old Flux store for whois data

#### Testing instructions

* Open the Contact edit form and the Contact section - make sure that it is loading correctly. Also manually refresh and check that it is loading correctly. Also add `?flags=force-sympathy` at the end and refresh to verify that the placeholder is shown until the data is actually loaded.
